tket2-py/test/test_guppy.py

```diff
@@ -9,8 +10,46 @@
 from guppylang.prelude.builtins import py
 from guppylang.prelude.quantum import measure, phased_x, qubit, rz, zz_max
 
+import pytket
 
-def test_load_compiled_module():
+
+def test_load_pure_circuit():
+    module = GuppyModule("test")
+    module.load(quantum)
+
+    @guppy(module)
+    @no_type_check
+    def my_func(
+        q0: qubit,
+        q1: qubit,
+    ) -> tuple[qubit, qubit]:  # pragma: no cover
+        q0 = phased_x(q0, py(math.pi / 2), py(-math.pi / 2))
```

Comment:
tket1 has angles in fractions of pi, are these angles converted correctly?

Reply:
Good catch. That was missing in the encoder/decoder.

Now we translate between radians and half-turns when converting hugrs to pytket circuits.